leval/rewriter_evaluator.py

```diff
@@ -16,6 +16,7 @@
     "in",
     "not",
     "or",
+    "is",
```

Comment:
Nit: can you keep this set sorted?
```suggestion
    "in",
    "is",
    "not",
    "or",
```